### ISSUE TITLE
Send Later tab keeps reappearing in the Inbox, even after removing

### ISSUE THREAD

**Issue by jcmuaz** [Extended-Thunder/send-later#460]
I'm just wondering how to permanently remove it from my Inbox folder. It keeps reappearing when I restart Tbird. I normally just have the Send Later column only in my Drafts folder. 

I'm using T-bird release v.102.2.2 (64 bit) on Win 10. I have Send Later v9.1.1. 
Thx!

**Comment by adorebo** [comment]
+1 On this one. 

The send later column appears on other than drafts folders only when opening Thunderbird, 
if you change folders it's then behaving as expected (only shows send later tab in drafts)

So just an issue when opening for me 
(Thunderbird v.102.3.0 (64 bit) on Win 10. Send Later v9.1.11)

Great software congrats it's useful !!

**Comment by adorebo** [comment]
Just found a way to correct this while waiting for a fix, the "send later" tab just have to be in another position than the first tab (to the left) and this problem won't occur when opening Thunderbird.

**Comment by jcmuaz** [comment]
> Just found a way to correct this while waiting for a fix, the "send later" tab just have to be in another position than the first tab (to the left) and this problem won't occur when opening Thunderbird.

I tried that, but unfortunately it didn't work. Just to make sure I followed your method, you mean that you moved the Send Later column tab in the Inbox folder, correct?

**Comment by jcmuaz** [comment]
I just noticed something new. Whatever folder was highlighted when I quit Thunderbird will be the one that it opens to when restarting. That fold will have the Send Later column in it, but if I move to another folder and then back, it will be gone. I've tried this a few times and it seems consistent. So, when I close out T-bird, I always make sure I'm highlighting the main email account name (right at the top of the folder list).
